cond/cond_test.go

```diff
@@ -393,7 +393,7 @@ func TestCond_Signal_random(t *testing.T) {
 		t.Fatal("test timeout")
 	}
 
-	assert.InDelta(t, 0.5, float64(successes)/float64(threads), 0.2, "roughly half of the threads should succeed")
+	assert.InDelta(t, 0.5, float64(successes)/float64(threads), 0.3, "roughly half of the threads should succeed")
```

Comment:
If your concern this is increasing the tolerance too much, could we make it tighter? Like 0.23 and explain with the additional decimal place? 🤷